### ISSUE TITLE
Redis sentinel

### ISSUE THREAD

**Issue by smsalert-mobi** [laminas/laminas-cache-storage-adapter-redis#77]
### Feature Request

|    Q        |   A
|------------ | ------
| New Feature | yes
| RFC         | yes
| BC Break    | no

#### Summary

Redis Sentinel provides high availability for Redis. In practical terms this means that using Sentinel you can create a Redis deployment that resists without human intervention certain kinds of failures.

Redis Sentinel also provides other collateral tasks such as monitoring, notifications and acts as a configuration provider for clients.

In a nutshell Redis Sentinel need to be queried in order to find out the actual redis master. 

More details can be found on the PHP extension documentation https://github.com/phpredis/phpredis/blob/develop/sentinel.md

**Comment by boesing** [comment]
Hey there, thanks for bringing this to my attention. as of now, I do not have the resources to implement this. if you have a project and experience with this Kind of setup, feel free to provide a patch implementing a PoC. I'll be happy to merge that, but wont work on it by myself.
